upgrading/automated_upgrades.adoc

```diff
@@ -629,6 +629,14 @@ xref:customizing-node-upgrades[Customizing Node Upgrades], continue running the
 *_upgrade_nodes.yml_* playbook until all nodes are upgraded.
 // tag::automated_upgrade_after_reboot[]
 
+.. If you have changed the cluster image source registry `oreg_url` for this upgrade 
```

Comment:
@jcpowermac, this looks good, but I'd change the text a little. Does this sound right to you? 

> If you modified the `oreg_url` parameter to change the cluster image registry location, you must run the `imageconfig` playbook to update the image location:

Reply:
@kalexand-rh yep that sounds good.  I sent an update for the change.